tests/templates/kuttl/oidc/login.py

```diff
@@ -0,0 +1,106 @@
+), "Redirection to the Airflow home page expected"
+
+# Open the user information page in Airflow
+userinfo_page = session.get("http://airflow-webserver:8080/users/userinfo/")
+
+assert userinfo_page.ok, "Retrieving user information failed"
+assert (
+    userinfo_page.url == "http://airflow-webserver:8080/users/userinfo/"
+), "Redirection to the Airflow user info page expected"
+
+# Expect the user data provided by Keycloak in Airflow
+userinfo_page_html = BeautifulSoup(userinfo_page.text, "html.parser")
+table_rows = userinfo_page_html.find_all("tr")
+user_data = {tr.find("th").text: tr.find("td").text for tr in table_rows}
+
+assert (
+    user_data["First Name"] == "Jane"
+), "The first name of the user in Airflow should match the one provided by Keycloak"
+assert (
+    user_data["Last Name"] == "Doe"
+), "The last name of the user in Airflow should match the one provided by Keycloak"
+assert (
+    user_data["Email"] == "jane.doe@stackable.tech"
+), "The email of the user in Airflow should match the one provided by Keycloak"
+
+# TODO Use different OIDC providers (currently only Keycloak is
+# supported)
+#
+# It would be beneficial if the second OAuth provider keycloak2 could
+# also be tested. This would ensure that the Airflow configuration is
+# correct. The problem is that the Flask-AppBuilder (and hence Airflow)
+# do not support multiple OAuth providers with the same name. But
+# keycloak1 and keycloak2 use the same name, namely "keycloak":
+#
+#  OAUTH_PROVIDERS = [
+#    { 'name': 'keycloak',
+#      'icon': 'fa-key',
+#      'token_key': 'access_token',
+#      'remote_app': {
+#        'client_id': os.environ.get('OIDC_728D9B504A6E9A10_CLIENT_ID'),
+#        'client_secret': os.environ.get('OIDC_728D9B504A6E9A10_CLIENT_SECRET'),
+#        'client_kwargs': {
+#          'scope': 'email openid profile'
+#        },
+#        'api_base_url': 'https://keycloak1.kuttl.svc.cluster.local:8443/realms/test1/protocol/',
+#        'server_metadata_url': 'https://keycloak1.kuttl.svc.cluster.local:8443/realms/test1/.well-known/openid-configuration',
+#      },
+#    },
+#    { 'name': 'keycloak',
+#      'icon': 'fa-key',
+#      'token_key': 'access_token',
+#      'remote_app': {
+#        'client_id': os.environ.get('OIDC_607BA683B09BC0B8_CLIENT_ID'),
+#        'client_secret': os.environ.get('OIDC_607BA683B09BC0B8_CLIENT_SECRET'),
+#        'client_kwargs': {
+#          'scope': 'email openid profile'
+#        },
+#        'api_base_url': 'https://keycloak2.kuttl.svc.cluster.local:8443/realms/test2/protocol/',
+#        'server_metadata_url': 'https://keycloak2.kuttl.svc.cluster.local:8443/realms/test2/.well-known/openid-configuration',
+#      },
+#    }
+#    ]
+#
+# This name is set in the operator and cannot be changed. The reason is
+# that the name is also used in Flask-AppBuilder to determine how the
+# user information must be interpreted.
+#
+# Airflow actually shows two "Sign In with keycloak" buttons in this
```

Comment:
Ah, OK, that answers my earlier question, but this will definitely be confusing for users.